pennylane_lightning/lightning_tensor/lightning_tensor.py

```diff
@@ -71,6 +71,7 @@
     {
         "Identity",
         "BasisState",
+        "MPSPrep",
```

Comment:
Is `StatePrep` missed here?

Reply:
Same question.

Reply:
I am not sure why `StatePrep` is not here. I checked the full history of the file and never was added to `_operation` list :thinking:

Reply:
Let's get back to this in a followup PR -- I have this on my list for Q1